index.html

```diff
@@ -7,6 +7,32 @@
     <title>merge-homeworks</title>
 </head>
 <body>
-    
+    <table width="100%" border="1" cellpadding="12" style="background-color: #1E352F;">
+        <thead align="left" style="background-color: #A6C36F;">
+            <th>Name:</th>
+            <th>Repo:</th>
+            <th>Preview:</th>
```

Comment:
```suggestion
            <tr>
                <th>Name:</th>
                <th>Repo:</th>
                <th>Preview:</th>
            </tr>
```
теги `<td>` та `<th>` повинні бути всередині `<tr>`